source/tools/cdk-integ-tools/lib/canonicalize-assets.ts

```diff
@@ -39,6 +41,24 @@
     ]);
   }
 
+  function checkIgnoreList(functionName: string): boolean {
+    for (const funcPrefix of LIST_OF_IGNORE_LAMBDA_PREFIX) {
+      if (functionName.startsWith(funcPrefix)) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  for (const [resourceName, resourceValue] of Object.entries(template?.Resources || {})) {
```

Comment:
This code could be self documenting if we moved this block out to a function:

```
function HideIgnoredResources(entries) {
    if (checkIgnoreList(resourceName)) {
      stringSubstitutions.push([
        (resourceValue as any).Properties.Code.S3Key,
        'SomeHash.zip'
      ])
    }
}
```

* needs better types on the declaration
* Probably some syntax issues in my version. :-)